Formula/dafny.rb

```diff
@@ -1,10 +1,10 @@
 class Dafny < Formula
   desc "Verification-aware programming language"
   homepage "https://github.com/dafny-lang/dafny/blob/master/README.md"
-  url "https://github.com/dafny-lang/dafny/archive/v2.3.0.tar.gz"
-  sha256 "ea7ae310282c922772a46a9a85e2b4213043283038b74d012047b5294687d168"
+  url "https://github.com/dafny-lang/dafny/archive/v3.0.0.tar.gz"
+  sha256 "5e9af6ca59c3329cd93d792bf9890c55c68c4f656afb19c85d1c44b0c7989fc2"
   license "MIT"
-  revision 3
+  revision 4
```

Comment:
```suggestion
```
You can remove this now, since this is a version bump.

Reply:
Done